src/lib.rs

```diff
@@ -30,11 +33,16 @@ hypher = { version = "0.1", default-features = false, features = ["english"] }
 ```
 */
 
+#![cfg_attr(not(test), no_std)]
 #![forbid(unsafe_code)]
 #![deny(missing_docs)]
 
-use std::fmt::{self, Debug, Formatter};
-use std::iter::FusedIterator;
+#[cfg(any(feature = "alloc", test))]
+#[cfg_attr(not(test), macro_use)]
```

Comment:
Also, which macro is this using?

Reply:
The `vec!` macro. It was only used once, so I just replaced it with `alloc::vec!`